packages/extension-browser/CONTRIBUTING.md

```diff
@@ -228,3 +260,9 @@ import NewParser from '@hint/parser-new';
         ]
     };
 ```
+
+<!-- Link labels -->
+
+[ff addon]: https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/
```

Comment:
The steps provided here are slightly out-of-date. Specifically they don't call out the need to click on "This Firefox" before the "Load Temporary Add-on..." button will be available.

Reply:
I know, but that's the official docs. I got redirected there from MDN so one can hope they'll update the information eventually. If you have a better (official) link please let me know!

Reply:
Perhaps @digitarald knows of a better link (or who to ask to update the docs)?

Reply:
I can file an issue for the page, as it just recently launched. It is the right link for sure.